src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -100,9 +133,9 @@ describe('ReviewScreen', () => {
       expect(queryByText('fiatConnectReviewScreen.transactionDetails')).toBeTruthy()
       expect(queryByText('fiatConnectReviewScreen.cashIn.transactionDetailsAmount')).toBeTruthy()
       expect(queryByTestId('txDetails-total/value')?.children).toEqual(['', '$', '100.00'])
-      expect(queryByTestId('txDetails-converted/value')?.children).toEqual(['', '$', '98.94'])
-      expect(queryByTestId('txDetails-fee/value')?.children).toEqual(['', '$', '1.06'])
-      expect(queryByTestId('txDetails-exchangeRate/value')?.children).toEqual(['', '$', '0.9894'])
+      expect(queryByTestId('txDetails-converted/value')?.children).toEqual(['', '$', '99.15'])
+      expect(queryByTestId('txDetails-fee/value')?.children).toEqual(['', '$', '0.84'])
+      expect(queryByTestId('txDetails-exchangeRate/value')?.children).toEqual(['', '$', '0.9915'])
```

Comment:
are these values changing because of different mock values for fee estimation?

Reply:
because of different values for the exchange rate